script.js

```diff
@@ -35,7 +35,7 @@ var FileDownloader = {
     var rawUrlNode = document.querySelector('#raw-url');
     if (rawUrlNode && !document.querySelector('#download-btn')) {
       var fileName = document.querySelector('.breadcrumb .final-path').textContent;
-      rawUrlNode.parentNode.innerHTML = "<a download='"+fileName+"' href='" + rawUrlNode.href + "' class='minibutton' id='download-btn'>Download</a>" + rawUrlNode.parentNode.innerHTML;
+      rawUrlNode.parentNode.innerHTML = "<a download='"+fileName+"' href='" + rawUrlNode.href + "' class='btn btn-sm' id='download-btn'>Download</a>" + rawUrlNode.parentNode.innerHTML;
```

Comment:
Using DOM would be even better but it's not the issue here, i just wanted to fix the broken display :
![issue](https://cloud.githubusercontent.com/assets/5783476/6797997/2528fb9a-d209-11e4-8f55-8a735a295fbc.png)